src/Uno.UWP/Devices/Geolocation/Geolocator.cs

```diff
@@ -77,11 +80,15 @@ public PositionAccuracy DesiredAccuracy
 				{
 					//force set DesiredAccuracy so that its ActualDesiredAccuracyInMeters rule is applied
 					DesiredAccuracy = DesiredAccuracy;
+#if __ANDROID__
+					OnDesiredAccuracyInMetersChanged();
+#endif
 				}
 			}
 		}
 
```

Comment:
use a partial method:

```suggestion
					OnDesiredAccuracyInMetersChanged();
				}
			}
		}
		
		partial void OnDesiredAccuracyInMetersChanged();
```

Reply:
I tried it. And about 5 minutes later, I got (from intellisense) `Feature 'extended partial methods' is not available in C# 8.0. Please use language version 9.0 or greater.`
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/cs8400
I don't know if I can switch to version 9.0 on my side, and besides that, I don't know how to change it :)

Reply:
Can it be left in form of `#if __ANDROID__` ?

Reply:
It certainly means that the partial method is not located at the right place, try changing the scope. Partial methods are supported since very early version of C#.

Reply:
I found it - "private partial" is not allowed :)